actor-server/actor-frontend/src/main/scala/im/actor/server/frontend/SessionClient.scala

```diff
@@ -211,7 +213,7 @@ private[frontend] final class SessionClient(sessionRegion: SessionRegion)
       } else {
         unpack(mbBits) match {
           case Success(rawBits) ⇒
-            sessionRegion.ref ! SessionEnvelope(authId, sessionId).withHandleMessageBox(HandleMessageBox(ByteString.copyFrom(rawBits.toByteBuffer)))
+            sessionRegion.ref ! SessionEnvelope(authId, sessionId).withHandleMessageBox(HandleMessageBox(ByteString.copyFrom(rawBits.toByteBuffer), Option(remoteAddr)))
```

Comment:
I understand that, but Option(remoteAddr) means that remoteAddr can be null (Option(null) == None), why not Some(remoteAddr)?